Keeran10/CurriculumManagementSystem
# Trends Feature Front-end + Heuristics

To verify:
1) Login to CMS.
2) Click on "Create new request".
3) Select a course.
4) Click on "Check Trends" beside impact-statement.

This PR completes issue #262 #263. Although 262 still needs more work as the heuristics are still not as accurate as it should be.
